### ISSUE TITLE
Uncaught Authorization Required

### ISSUE THREAD

**Issue by Sunnepah** [p-baleine/atom-twitter-client#8]
Reproducing steps:

[Packages] -> [Twitter] -> [Search], then enter the keyword.
# 

**Atom Version**: 1.0.0
**System**: Mac OS X 10.10.4
**Thrown From**: [atom-twitter-client](https://github.com/p-baleine/atom-twitter-client) package, v0.1.0
### Stack Trace

Uncaught Authorization Required

```
At /Users/sunnepah/.atom/packages/atom-twitter-client/lib/atom-twitter-timeline-view.coffee:20

undefined
```
### Commands

```
  2x -6:16.6.0 atom-twitter:home (atom-text-editor.editor.is-focused)
  2x -1:02.9.0 deprecation-cop:view (atom-workspace.workspace.scrollbars-visible-always.theme-solarized-dark-syntax.theme-one-dark-ui)
     -0:35.4.0 atom-twitter:home (atom-pane.pane.active)
     -0:15.6.0 atom-twitter:search (atom-pane.pane.active)
     -0:05.8.0 core:confirm (atom-text-editor.editor.mini.is-focused)
```
### Config

``` json
{
  "core": {
    "themes": [
      "one-dark-ui",
      "solarized-dark-syntax"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
atom-twitter-client, v0.1.0

# Dev
No dev packages
```

**Comment by p-baleine** [comment]
@Sunnepah  Thanks to report.
This issue was fixed at 7d4390932c4d1b0eda5d9cfe9276a2f6139bacfc.
Please update this package.

**Comment by Sunnepah** [comment]
@p-baleine Thanks. It works fine now. cheers!
